mysite/general/secretPath.py

```diff
@@ -9,35 +9,72 @@
 
 skillsToReview_RightHand = ["Mining", "Choppin", "Fishing", "Catching", "Trapping"]  #, "Worship"]
 
-def getHandsAdviceGroup(maestros, beginners):
+def getHandsAdviceGroupCatchUp(maestros, beginners):
+    # This function returns the skills maestros aren't best in.
     janky_skills = maestros_goal_levels(maestros)
-    tier = f"{len(janky_skills) or 6}/{len(skillsToReview_RightHand)}"
+    zero_count = sum(1 for skill, (_, _, next_highest_non_maestro_char) in janky_skills.items() if not next_highest_non_maestro_char)
+    if zero_count == 5:
```

Comment:
Instead of hard-coding 5, I'd prefer to use the length of the skillsToReview_RightHand list. That way it updates automatically in the future if more skills get added or removed.